plasmapy/classes/plasma_base.py

```diff
@@ -0,0 +1,26 @@
+from collections import OrderedDict
+from abc import ABC
+
+# GenericPlasma subclass registry
+PLASMA_CLASSES = OrderedDict()
```

Comment:
I know this is how it's done in SunPy, but actually I would not have this as a module level constant. I would just have it as a class attribute on `GenericPlasmaRegistrar`

Reply:
Yep, good idea.